pkg/gds/admin_test.go

```diff
@@ -609,39 +609,92 @@ func (s *gdsTestSuite) TestRetrieveVASP() {
 	rep := s.doRequest(a.RetrieveVASP, c, w, nil)
 	require.Equal(http.StatusNotFound, rep.StatusCode)
 
-	charlieID := s.fixtures[vasps]["charliebank"].(*pb.VASP).Id
+	hotel := s.fixtures[vasps]["hotel"].(*pb.VASP)
 
 	// Retrieve a VASP that exists
-	request.path = "/v2/vasps/" + charlieID
+	request.path = "/v2/vasps/" + hotel.Id
 	request.params = map[string]string{
-		"vaspID": charlieID,
+		"vaspID": hotel.Id,
 	}
 	actual := &admin.RetrieveVASPReply{}
 	c, w = s.makeRequest(request)
 	rep = s.doRequest(a.RetrieveVASP, c, w, actual)
 	require.Equal(http.StatusOK, rep.StatusCode)
 	expected := &admin.RetrieveVASPReply{
-		Name:     "CharlieBank",
+		Name:     "Hotel Corp",
 		Traveler: false,
 		AuditLog: []map[string]interface{}{
 			{
 				"current_state":  pb.VerificationState_SUBMITTED.String(),
 				"description":    "register request received",
 				"previous_state": pb.VerificationState_NO_VERIFICATION.String(),
 				"source":         "automated",
-				"timestamp":      "2021-08-31T12:36:27Z",
+				"timestamp":      "2021-06-17T11:12:23Z",
+			},
+			{
+				"current_state":  pb.VerificationState_EMAIL_VERIFIED.String(),
+				"description":    "completed email verification",
+				"previous_state": pb.VerificationState_SUBMITTED.String(),
+				"source":         "automated",
+				"timestamp":      "2021-06-21T14:34:49Z",
+			},
+			{
+				"current_state":  pb.VerificationState_PENDING_REVIEW.String(),
+				"description":    "review email sent",
+				"previous_state": pb.VerificationState_EMAIL_VERIFIED.String(),
+				"source":         "automated",
+				"timestamp":      "2021-07-01T20:59:04Z",
+			},
+			{
+				"current_state":  pb.VerificationState_REVIEWED.String(),
+				"description":    "registration request received",
+				"previous_state": pb.VerificationState_PENDING_REVIEW.String(),
+				"source":         "admin@rotational.io",
+				"timestamp":      "2021-08-10T21:37:14Z",
+			},
+			{
+				"current_state":  pb.VerificationState_ISSUING_CERTIFICATE.String(),
+				"description":    "issuing certificate",
+				"previous_state": pb.VerificationState_REVIEWED.String(),
+				"source":         "automated",
+				"timestamp":      "2021-08-25T18:03:15Z",
+			},
+			{
+				"current_state":  pb.VerificationState_VERIFIED.String(),
+				"description":    "certificate issued",
+				"previous_state": pb.VerificationState_ISSUING_CERTIFICATE.String(),
+				"source":         "automated",
+				"timestamp":      "2021-10-21T15:52:08Z",
```

Comment:
Only our "verified" VASP fixtures have certificates so I had to change the fixture here, which also has a different audit log.